<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yuv_dlg</class>
 <widget class="QDialog" name="yuv_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QComboBox" name="fmt_cbox">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>28</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>YUV420</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>YUV422</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>YUV444</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>YUY2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>UYVY</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>YV12</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>28</height>
               </size>
              </property>
              <property name="text">
               <string>FPS</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fps_edit">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QComboBox" name="res_cbox">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>28</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>QCIF</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CIF</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SD</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>HD</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>FHD</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Custom</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="xres_edit">
              <property name="minimumSize">
               <size>
                <width>45</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>45</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>10</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>10</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="yres_edit">
              <property name="minimumSize">
               <size>
                <width>45</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>45</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="file_btn">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="path_edit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="start_btn">
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
